quickwit/quickwit-metastore/src/metastore/postgres/queries/shards/open.sql

```diff
@@ -1,5 +1,5 @@
-INSERT INTO shards(index_uid, source_id, shard_id, leader_id, follower_id)
-    VALUES ($1, $2, $3, $4, $5)
+INSERT INTO shards(index_uid, source_id, shard_id, leader_id, follower_id, publish_token)
+    VALUES ($1, $2, $3, $4, $5, $6)
```

Comment:
how is the publish token handled for ingest v2 today?

Reply:
from what I understood, the token is not acquired when openeing the shard in ingest v2 because this operation is performed by the control plane. The ingest sources later call acquire to assign a publish token.